CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+
+Help us keep Appwrite open and inclusive. Please read and follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
+
+## Getting Started
+
+Install using composer:
+```bash
+composer require appwrite/php-runtimes
+```
+
+```php
+<?php
+
+require_once '../vendor/autoload.php';
+
+use Appwrite\Runtimes\Runtimes;
+
+$runtimes = new Runtime();
+
+// returns all supported cloud function runtimes
+Runtimes::getAll();
+```
+
+## Adding a new Runtime
+
+After the Docker image is created, this must be added to the main class of this library.
+
+References to this must be added to the constructor of `src/Runtimes/Runtimes.php`.
+
+Example:
+
+```php
+$dotnet = new Runtime('dotnet', '.NET');
+$dotnet->addVersion('5.0', 'mcr.microsoft.com/dotnet/runtime:5.0-alpine', 'appwrite/env-dotnet-5.0:1.0.0', [System::X86, System::ARM]);
+$runtimes[] = $dotnet;
+```
+
+## Function Runtimes Checklist
+
+The following checklist aims to ensure that a function runtime gets added successfully
+
+- [ ] Implement the runtime in [open-runtimes/open-runtimes](https://github.com/open-runtimes/open-runtimes) (you can find the tutorial [here](https://github.com/open-runtimes/open-runtimes/blob/main/docs/add-runtime.md))
+  - [ ] Prepare the files for your new runtime
+    - [ ] Dockerfile
+    - [ ] Readme
+  - [ ] Write the runtime
+    - [ ] Initialize a web server
+      - [ ] Set Port 3000
+      - [ ] Bind IP 0.0.0.0
+      - On each POST Request
+        - [ ] Check that the `x-internal-challenge header` matches the `INTERNAL_RUNTIME_KEY` environment variable
+        - [ ] Decode the executor's JSON POST request
+    - [ ] Create Request Class
+      - [ ] Fields
+        - [ ] env
+        - [ ] payload
+        - [ ] headers
+    - [ ] Create Response Class
+      - [ ] Functions
+        - [ ] send(string)
+        - [ ] json(object)
+    - [ ] Execute the function
+      - [ ] Add `try catch` block for error handling
+  - [ ] Write the Dockerfile
+    - [ ] Add the Docker image you want to base your runtime off
+    - [ ] Create the folders you'll use
+    - [ ] Copy your source code and set working directory
+    - [ ] Add execute permissions for any scripts
+      - [ ] `build.sh`
+      - [ ] `start.sh`
+    - [ ] Use `RUN` commands for necessary dependencies (if needed)
+    - [ ] Expose port 3000
+    - Add a `CMD` command for your launch script (i.e. `start.sh`)
+  - [ ] Build your Docker image and add it to the script files
+  - [ ] Add test
+    - [ ] Create a PHP file named by the language   
+    - [ ] Create a new folder in `./tests` by the name of your runtime
+      - [ ] Create a function (steps)
+        - [ ] Decode the payload as JSON
+        - [ ] Set a string variable called `id` to the value of the `id` key in the payload or to `1` if it doesn't exist
+        - [ ] Fetch `https://jsonplaceholder.typicode.com/todos/$id` with an HTTP Client installed from your language's package manager using the `id` variable
+        - [ ] return `res.json`
```

Comment:
Again, let's link to some existing test (JS probably, that is easy to follow). They just got to do the same but in their language.